<?xml version="1.0"?>
<launch>   
    <node name="mocapnet_rosnode" pkg="mocapnet_rosnode" type="run_it.sh" output="screen" cwd="node" respawn="true">
      <param name="name" value="mocapnet_rosnode" /> 
      <param name="fromRGBTopic" value="/camera/rgb/image_rect_color" /> 
      <param name="fromRGBTopicInfo" value="/camera/rgb/camera_info" />

      <param name="tfRoot" value="map" /> 
      <param name="tfTargetBVHFilename" value="dataset/headerWithHeadAndOneMotion.bvh" />
      <param name="useSimple3DPointTF" value="0" /> 
      <param name="publishCameraTF" value="1" /> 
      <param name="cameraXPosition" value="0.0" /> 
      <param name="cameraYPosition" value="0.0" /> 
      <param name="cameraZPosition" value="0.0" /> 
      <param name="cameraRoll" value="90.0" /> 
      <param name="cameraPitch" value="0.0" /> 
      <param name="cameraYaw" value="0.0" /> 

      <param name="joint2DEstimator" value="forth" />  

      <param name="useCPUOnlyForMocapNET" value="1" /> 
      <param name="useCPUOnlyFor2DEstimator" value="0" /> 
      <param name="multithreaded" value="0" /> 
 
      <param name="useHierarchicalCoordinateDescent" value="1" /> 
      <param name="hierarchicalCoordinateDescentLearningRate" value="0.01" /> 
      <param name="hierarchicalCoordinateDescentSpring" value="20.0" /> 
      <param name="hierarchicalCoordinateDescentIterations" value="5" /> 
      <param name="hierarchicalCoordinateDescentEpochs" value="30" />
    </node>   
</launch>
